### PR TITLE
[ko] Update outdated in dev-1.22-ko.2(36-37)

### PR DESCRIPTION
**Pull request by seokho-son**
Refs #30000 
Fix tasks: M36-M37

- M36. content/en/docs/concepts/workloads/pods/_index.md | 9(+XS) 2(-)
- M37. content/en/docs/concepts/workloads/pods/init-containers.md | 10(+S) 6(-)

### CODE REVIEW

File: content/ko/docs/concepts/workloads/pods/init-containers.md [hunk 1]
```diff
@@ -32,12 +32,14 @@ weight: 40
...
 컨테이너를 초기화 컨테이너로 지정하기 위해서는,
-파드 스펙에 앱 `containers` 배열과 나란히 `initContainers` 필드를
-[컨테이너](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#container-v1-core)
-타입 오브젝트들의 JSON 배열로서 추가한다.
-초기화 컨테이너의 상태는 `.status.initContainerStatuses` 필드를
-통해서 컨테이너 상태 배열로 반환된다
-(`.status.containerStatuses` 필드와 유사하게).
+[파드 스펙](/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec)에 `initContainers` 필드를
+`container` 항목(앱 `container` 필드 및 내용과 유사한)들의 배열로서 추가한다.
+[컨테이너]](/docs/reference/kubernetes-api/workload-resources/pod-v1/#Container)에 대한 더 상세한 사항은
```
Comment (ClaudiaJKang) [inline content/ko/docs/concepts/workloads/pods/init-containers.md:37]:
```suggestion
[컨테이너](/docs/reference/kubernetes-api/workload-resources/pod-v1/#Container)에 대한 더 상세한 사항은
```

`]` 삭제가 필요할 것 같습니다~

**Review by ClaudiaJKang — COMMENTED** [review]
/assign

리뷰 완료했습니다~

**Comment by seokho-son** [comment]
@ClaudiaJKang 감사합니다! 업데이트 및 스쿼싱 완료하였습니다.

**Pull request merged**